### DISCUSSION TITLE
Ray on WSL2

### DISCUSSION THREAD

**Discussion by mhluzi** [spatie/laravel-ray#107] [Q&A] [answered]
I am trying to get Ray working on my Laravel project (using WSL2) but am not having any luck.

I have installed the package and added the test line:

        ray('My first ray call');

but nothing appears in the Ray app.

The call is being passed to the LaravelRay::class:

![2021-01-18_14-51-29](https://user-images.githubusercontent.com/7857973/104867640-35622600-59a6-11eb-9f1b-38bdf730412e.png)

so I am wondering if it is a configuration issue on my side.

My ray config is:

    'enable' => env('RAY_ENABLED', true),
    'send_log_calls_to_ray' => env('SEND_LOG_CALLS_TO_RAY', true),
    'send_dumps_to_ray' => env('SEND_DUMPS_TO_RAY', true),
    'host' => env('RAY_HOST', '127.0.0.1:9090'),
    'port' => env('RAY_PORT', 23517),
    'remote_path' => env('RAY_REMOTE_PATH', '/home/ubuntu2004/mysite'),
    'local_path' => env('RAY_LOCAL_PATH', '\\wsl$\Ubuntu-20.04\home\ubuntu2004\mysite'),

Is there something else I can try or do to get this to work. (I have the paid version)

Thanks!

**Comment by freekmurze** [comment, marked as answer]
We'll track this here: https://github.com/spatie/ray/issues/191

**Reply by mhluzi** [reply]
Thanks!
